kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/tasks/graal/ios/downloadIOSJdkStaticLibraries.kt

```diff
@@ -0,0 +1,44 @@
+package godot.gradle.tasks.graal.ios
+
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.tasks.TaskProvider
+import org.jetbrains.kotlin.gradle.internal.ensureParentDirsCreated
+import java.net.URL
+
+fun Project.downloadIOSJdkStaticLibraries(): TaskProvider<out Task> = tasks
+    .register("downloadIOSJdkStaticLibraries") {
+        with(it) {
+            group = "godot-kotlin-jvm-internal"
+            description = "INTERNAL TASK ! Downloads ios static JDK libraries."
+
+            doLast {
+                val iosJdkDirectory = buildDir
+                    .resolve("libs")
+                    .resolve("ios")
+                    .resolve("ios-jdk")
+                    .resolve("21")
+
+                val iosGraalJdkReleaseBaseLink = "https://github.com/utopia-rise/ios-graal-jdk-21/releases/download/v23.1.3-21-b33/"
```

Comment:
Could we move this download url (or at least the version) to the `gradle/[libs.versions.toml](https://github.com/utopia-rise/godot-kotlin-jvm/blob/master/kt/gradle/libs.versions.toml)` and use it through the build props (https://github.com/utopia-rise/godot-kotlin-jvm/blob/a4c5385efb08621d19a29f0db3cc61b132b2fc54/kt/plugins/godot-gradle-plugin/build.gradle.kts#L42)
Like we do for our own version and the kotlin version